pkg/provider/provider_acceptance_test.go

```diff
@@ -154,6 +154,79 @@ func TestAcc_Provider_configureClientOnceSwitching(t *testing.T) {
 	})
 }
 
+func TestAcc_Provider_useNonExistentDefaultParams(t *testing.T) {
+	t.Setenv(string(testenvs.ConfigureClientOnce), "")
+
+	nonExisting := "NON-EXISTENT"
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck: func() {
+			acc.TestAccPreCheck(t)
+			testenvs.AssertEnvNotSet(t, snowflakeenvs.User)
+			testenvs.AssertEnvNotSet(t, snowflakeenvs.Password)
+		},
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config:      providerConfigWithRole(testprofiles.Default, nonExisting),
+				ExpectError: regexp.MustCompile("Role 'NON-EXISTENT' specified in the connect string does not exist or not authorized."),
+			},
+			{
+				Config:      providerConfigWithWarehouse(testprofiles.Default, nonExisting),
+				ExpectError: regexp.MustCompile("The requested warehouse does not exist or not authorized."),
+			},
+			{
+				Config: providerConfigWithWarehouseAndDisabledValidation(testprofiles.Default, nonExisting),
```

Comment:
what's the expectation in this last step?

Reply:
Done.